Link to template for proposals (#82)
tc39/how-we-work: champion.md

### Managing a GitHub repository

> Each staged proposal should have a GitHub repository, included in the tc39 organization once the proposal is presented to the committee. A good place to start is by copying (not forking) the [template-for-proposals](https://github.com/tc39/template-for-proposals#create-your-proposal-repo) repository. Your proposal's repository should include resources like the explainer, draft specification text, and sometimes draft implementations or example programs. The repository's issue tracker can be used to discuss design issues with the committee and the community. These proposal repositories are managed in accordance with [TC39's Code of Conduct](https://tc39.es/code-of-conduct/).

Reviewer: ```suggestion
Each staged proposal should have a GitHub repository, included in the tc39 organization once the proposal is presented to the committee. A good place to start is by clicking "use this template" button on [template-for-proposals](https://github.com/tc39/template-for-proposals#create-your-proposal-repo) repository. Your proposal's repository should include resources like the explainer, draft specification text, and sometimes draft implementations or example programs. The repository's issue tracker can be used to discuss design issues with the committee and the community. These proposal repositories are managed in accordance with [TC39's Code of Conduct](https://tc39.es/code-of-conduct/).
```

Copying is a bad idea now that template repos exist, and forking is no longer necessary for the same reason.

Author: Ah, neat. Updated this PR to just refer to that one; we should fix the documentation there.

Reviewer: Updated in the template repo.
